allen-cell-animated/cell-feature-explorer
# Fix/remove hard coded images

Problem
=======
What is the problem this work solves, including
[Link to story or ticket](https://aicsjira.corp.alleninstitute.org/browse/CFE-84)

Solution
========
Removed the imageMapping code

## Type of change
Please delete options that are not relevant.

* Bug fix (non-breaking change which fixes an issue)

Change summary:
---------------
* Removed code
* removed images themselves 

Steps to Verify:
----------------
1. npm start
2. load app, since the images are already in the db no changes should be noticeable 


Thanks for contributing!
